packages/xo-remote-parser/src/index.js

```diff
@@ -37,9 +37,11 @@ export const parse = string => {
     object.domain = domain
     object.username = username
     object.password = password
-  } else if (type === 's3') {
+  } else if (type === 's3' || type === 's3+http') {
     const parsed = new Url(string)
+    object.protocol = parsed.protocol === 's3:' ? 'https' : 'http'
     object.type = 's3'
+    object.region = parsed.hash.length === 0 ? undefined : parsed.hash.slice(1) // remove '#'
```

Comment:
Isn't the region already in the URL?

E.g.: `https://my-bucket.s3.us-west-2.amazonaws.com/puppy.png`, `us-west-2`?

Reply:
well, I discovered that the access point region and the storage region can be different, at AWS you could pay for extra transfer fees if they are not the same. For S3-compatible servers, their access point might be completely unrelated to region naming.